src/scss/components/_dataUpload.scss

```diff
@@ -13,35 +14,72 @@
         }
 
         h2,  h3{
+          display: flex;
+          justify-content: center;
+        }
+
+        @media only screen and (max-width: 375px) {
```

Comment:
Need to change max width to 500px, as this does not work on many mobile devices including mine:
![IMG_4139](https://user-images.githubusercontent.com/20331396/77210451-6a2bf500-6ad7-11ea-9037-4b3950707892.PNG)

Reply:
FIXED